js/buttons.js

```diff
@@ -0,0 +1,103 @@
+function addButtons() {
+  var buttonBox = waterUseViz.elements.buttonBox = svg.append('g')
```

Comment:
woah - what's the result of having the two equals? Do `buttonBox` and `waterUseViz.elements.buttonBox` become the d3 group selection?

Reply:
You can do this in R, too. Yes, the elements on the left and in the middle get set to the rightmost value. I'm just using it here so I don't have to spell out `waterUseViz.elements.buttonBox` everywhere in this function.